### ISSUE TITLE
Polish typeahead search (keyboard nav, styling)

### ISSUE THREAD

**Issue by tsmacdonald** [metabase/metabase#14584]
* Arrow keys and `Enter` should be able to select search results in the popover
* "No results" should look like something
* There should be a link to the full search page (`/search?q=foo`)

**Comment by cdeweyx** [comment]
Noting here that the following have been addressed: 
- Arrow keys and Enter should be able to select search results in the popover
- "No results" should look like something

Still open:
- There should be a link to the full search page (/search?q=foo)

**Comment by tsmacdonald** [comment]
@cdeweyx are we sold on

> There should be a link to the full search page (/search?q=foo)

being necessary? You can get there by pressing `Enter`. I originally opened this issue when search was way less polished and I was just brainstorming UX features that would help (note: I am not a designer :) )

**Comment by cdeweyx** [comment]
@tsmacdonald Definitely not necessary at the moment, but could be relevant in the future. Full page search isn't particularly useful right now so increasing awareness of it is first dependent on improving the page itself.

**Comment by tsmacdonald** [comment]
This landed in 48: https://github.com/metabase/metabase/issues/27982
